content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,556 @@
+---
+title: JavaScript SDKs
+description:
+  This article serves as a guide on how to get started using
+  Sharetribe's JavaScript SDKs.
+---
+
+import { Callout, Table, Steps, Tabs } from 'nextra/components';
+
+# Getting started with the JavaScript SDKs
+
+Sharetribe's
+[JavaScript Software Development Kit](https://sharetribe.github.io/flex-sdk-js/)
+(SDK) and
+[Integration SDK](https://sharetribe.github.io/flex-integration-sdk-js/)
+simplify interactions with our APIs. You can read more about features
+and functionalities in our
+[SDK GitHub Documentation](https://sharetribe.GitHub.io/flex-sdk-js/index.html).
+This guide covers
+
+- Installation
+- Initialization and instantiating the SDK
+- Creating a trusted SDK instance
+- Example use cases
+
+## Overview
+
+Our SDKs simplify making requests to the backend by handling
+authentication and providing convenient access to Sharetribe's APIs.
+They allow you to interact easily with
+
+- [Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html),
+  for general interactions with marketplace data
+- [Integration API](https://www.sharetribe.com/api-reference/integration.html),
+  for advanced operational use
+- [Asset Delivery API](https://www.sharetribe.com/api-reference/asset-delivery-api.html),
+  for delivering assets configured in Console
+
+You can find more information on API usage and guidelines for choosing
+which to work with in
+[this article](https://www.sharetribe.com/docs/concepts/api-sdk/marketplace-api-integration-api/#when-to-use-the-marketplace-api).
+
+There are two JavaScript SDKs you can use to access the APIs
+
+<br />
+<Table>
+  <thead>
+    <Table.Tr>
+      <Table.Th />
+      <Table.Th>Sharetribe SDK</Table.Th>
+      <Table.Th>Sharetribe Integration SDK</Table.Th>
+    </Table.Tr>
+  </thead>
+  <tbody>
+    <Table.Tr>
+      <Table.Td>**Access level**</Table.Td>
+      <Table.Td>Limited to user-scope and public data</Table.Td>
+      <Table.Td>Full marketplace (operator-level) access</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>**API access**</Table.Td>
+      <Table.Td>Marketplace API and Asset Delivery API</Table.Td>
+      <Table.Td>Integration API</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>**Required Parameters**</Table.Td>
```

Comment:
Comment: "parameters" is quite generic, many things can be parameters. I think a more descriptive word would be "Client credentials", because these are specific "parameters" that we use to authenticate the SDK